src/site/_includes/pattern-set.njk

```diff
@@ -46,110 +67,71 @@ pageScripts:
+      </div>
+      <div class="all-center">
+        {% Img
           height=220,
           width=330,
           src=patternSuite.hero,
           alt="Collection cover image",
           decoding="async",
-          class="w-masthead-path__image"
+          class="hero__content__img"
         %}
+      </div>
     </div>
   </div>
 </header>
 
-<hr>
+<div class="wrapper">
+  <div class="sidebar region flex-align-start flex-dir-rev flex-wrap-no">
+    {% include 'partials/toc-side.njk' %}
 
-<section class="w-layout-container--wide code-patterns-nav--top">
-  <nav class="code-patterns-nav__sets">
-    {% for setId, set in patterns.sets %}
-      {% if set.suite == patternSuite.id %}
-        {% if setId == currentPatternSet.id %}
-          <a class="selected" href="/patterns/{{set.id}}">{{set.title}}</a>
-        {% else %}
-          <a href="/patterns/{{set.id}}">{{set.title}}</a>
-        {% endif %}
-      {% endif %}
-    {% endfor %}
-  </nav>
-  <div class="code-patterns-nav__filter">
-    {# <input type="text" placeholder="filter"/> #}
-  </div>
-</section>
-
-<section class="w-layout-container--wide code-patterns-main">
-  <div>
-    {# Show patterns from current set (if set) or from the whole suite #}
-    {% if currentPatternSet %}
-      <h2 class="w-headline--two">{{ currentPatternSet.title }}</h2>
-      {{ content | safe }}
-      {% for patternId, pattern in patterns.patterns %}
-        {% if pattern.set == currentPatternSet.id %}
-          {{ codePattern(pattern) }}
-        {% endif %}
-      {% endfor %}
+    <div class="prose legacy-rollout">
+      {% include 'partials/toc-inner.njk' %}
+      {# Show patterns from current set (if set) or from the whole suite #}
+      {% if currentPatternSet %}
+        <h2>{{ currentPatternSet.title }}</h2>
+        {{ content | safe }}
+        {% for patternId, pattern in patterns.patterns %}
+          {% if pattern.set == currentPatternSet.id %}
+            {{ codePattern(pattern) }}
+          {% endif %}
+        {% endfor %}
 
-    {% else %}
+      {% else %}
 
-      {# Patterns in PatternSets that belong to the current suite #}
-      {% for patternSetId, patternSet in patterns.sets %}
-        {% if patternSet.suite == patternSuite.id %}
-          <h2 class="w-headline--two">{{ patternSet.title }}</h2>
-          {{ patternSet.rawContent | siteRender | safe }}
-          {% for patternId, pattern in patterns.patterns %}
-            {% if pattern.set == patternSetId %}
-              {{ codePattern(pattern) }}
+        {# Patterns in PatternSets that belong to the current suite #}
+        {% for patternSetId, patternSet in patterns.sets %}
+          {% if patternSet.suite == patternSuite.id %}
+            <h2>{{ patternSet.title }}</h2>
+            {{ patternSet.rawContent | siteRender | safe }}
+            {% for patternId, pattern in patterns.patterns %}
+              {% if pattern.set == patternSetId %}
+                {{ codePattern(pattern) }}
```

Comment:
List of patterns is malformed:
<img width="954" alt="Screenshot 2022-01-24 at 13 41 07" src="https://user-images.githubusercontent.com/1914261/150784682-45137839-0c1c-4f90-92ea-9c1d9d9ff470.png">

Reply:
I think I fixed this, but if you could check that'd be awesome. I think our computers are rendering this differently.